src/components/recursive_tree/recursive_tree.tsx

```diff
@@ -290,12 +312,14 @@ export class EuiRecursiveTree extends Component<
                         this.onChildrenKeydown(event, index)
                       }>
                       {node.children && this.isNodeOpen(node) ? (
+                        // @ts-ignore
                         <EuiRecursiveTree
                           items={node.children}
                           isCondensed={isCondensed}
                           showExpansionArrows={showExpansionArrows}
                           expandByDefault={this.state.expandChildNodes}
-                          aria-labelledby={this.state.treeID}
+                          isNested={true}
+                          {...label}
```

Comment:
Just to explain what's going on...

1. Above I check if the user passed in `aria-label` or `aria-labelledby`
2. I then create a new label of the same type, either building a string or a list of IDs
3. I put that in an object so I can easily spread either type into the component instead of having to conditionally render two different attributes